### DISCUSSION TITLE
Renovate suddenly stopped working due to `ERROR: Repository is not found`

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [renovatebot/renovate#21532] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

Renovate suddenly stopped working due to `ERROR: Repository is not found` in a specific repository.

- Repository: https://github.com/aquaproj/aqua-registry
- renovate configuration: https://github.com/aquaproj/aqua-registry/blob/b5542a0829b28421d58df578e6df9542ff3f1eff/renovate.json5
- Dependency Dashboard: https://github.com/aquaproj/aqua-registry/issues/1

This repository is a public repository. Renovate GitHub App is installed, so Renovate GitHub App should be able to access this repository.

Renovate Dashboard.

<img width="1057" alt="image" src="https://user-images.githubusercontent.com/13323303/232268783-bd52f072-d841-4a01-ac0e-63e892ecee50.png">

Renovate Log. https://app.renovatebot.com/dashboard#github/aquaproj/aqua-registry/1049664499

```
DEBUG: No dangling containers to remove
INFO: Repository started
{
  "renovateVersion": "35.40.0"
}
DEBUG: Using localDir: /mnt/renovate/gh/aquaproj/aqua-registry
DEBUG: PackageFiles.clear() - Package files deleted
DEBUG: initRepo("aquaproj/aqua-registry")
DEBUG: Using queue: host=api.github.com, concurrency=10
DEBUG: Caught initRepo error
{
  "err": {
    "message": "not-found",
    "stack": "Error: not-found\n    at Proxy.initRepo (/home/ubuntu/renovateapp/node_modules/renovate/dist/modules/platform/github/index.js:275:19)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async getPlatformConfig (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/init/apis.js:40:28)\n    at async initApis (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/init/apis.js:49:14)\n    at async initRepo (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/init/index.js:32:14)\n    at async Object.renovateRepository (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/index.js:46:18)\n    at async renovateRepository (/home/ubuntu/renovateapp/app/worker/index.js:260:26)\n    at async /home/ubuntu/renovateapp/app/worker/index.js:454:5"
  }
}
ERROR: Repository is not found
```

### Relevant debug logs.

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
That error appears to be coming from GitHub. Can you try suspending/unsuspending the app to see if that makes any difference?

**Reply by suzuki-shunsuke** [reply]
Thank you for your support.
I tried to suspend and unsuspend the GitHub App.

<img width="1014" alt="image" src="https://user-images.githubusercontent.com/13323303/232322738-e2c9339a-97c7-4598-a7ed-d1bc9045a182.png">

But there was no change.

<img width="966" alt="image" src="https://user-images.githubusercontent.com/13323303/232322636-d1dd0f5e-5e0a-418d-a37f-d6076a9a215f.png">

**Reply by suzuki-shunsuke** [reply]
Hmm. 
Currently Renovate jobs aren't run, but this issue may be a different issue from `Repository is not found`.

The other day Renovate stopped working in this repository. https://github.com/renovatebot/renovate/discussions/21410
The issue has been solved by splitting dependencies by schedule, but maybe the same issue recurred somehow. 🤔 

https://github.com/aquaproj/aqua-registry/blob/b5542a0829b28421d58df578e6df9542ff3f1eff/renovate.json5

**Reply by suzuki-shunsuke** [reply, marked as answer]
The issue seems to be caused by https://github.com/aquaproj/aqua-registry/pull/11534 so I reverted the change by https://github.com/aquaproj/aqua-registry/pull/11572 , then the issue has been solved.
Now Renovate is working fine.
